ui/apps/dashboard/src/components/navigation/index.tsx

```diff
@@ -63,6 +66,10 @@ const Navigation: FC<INavigationProps> = (props) => {
           </div>
           <div className={styles.right}>
             {/* extra components */}
+            <span className={styles.terminalIcon} onClick={() => onTerminalClick && onTerminalClick()}>
```

Comment:
for typescript, it can be shorten like `onTerminalClick?.()`
it will be compile like 
![image](https://github.com/user-attachments/assets/a9bf6290-6bb4-4e8b-a989-7713c110135f)